src/app/tests/suites/certification/Test_TC_SMCO_2_1.yaml

```diff
@@ -0,0 +1,156 @@
+# Copyright (c) 2023 Project CHIP Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: 4.2.1. [TC-SMCO-2.1] Attributes with DUT as Server
+
+PICS:
+    - SMOKECO.S
+
+config:
+    nodeId: 0x12344321
+    cluster: "Smoke CO Alarm"
+    endpoint: 1
+
+tests:
+    - label: "Commission DUT to TH"
+      cluster: "DelayCommands"
+      command: "WaitForCommissionee"
+      arguments:
+          values:
+              - name: "nodeId"
+                value: nodeId
+
+    - label: "TH reads from the DUT the ExpressedState attribute"
+      PICS: SMOKECO.S.A0000
+      command: "readAttribute"
+      attribute: "ExpressedState"
+      response:
+          constraints:
+              type: enum8
+              minValue: 0
+              maxValue: 8
+
+    - label: "TH reads from the DUT the SmokeState attribute"
+      PICS: SMOKECO.S.A0001
+      command: "readAttribute"
+      attribute: "SmokeState"
+      response:
+          constraints:
+              type: enum8
+              minValue: 0
+              maxValue: 2
```

Comment:
This does not match the test plan, as I said.  Please note the tracking test plan issue here.